src/solc_0.8/OperatorFilterer/contracts/OperatorFilterRegistrant.sol

```diff
@@ -0,0 +1,23 @@
+//SPDX-License-Identifier: MIT
+// solhint-disable-next-line compiler-version
+pragma solidity 0.8.2;
+
+import {IOperatorFilterRegistry} from "../interfaces/IOperatorFilterRegistry.sol";
+import "@openzeppelin/contracts-0.8/access/Ownable.sol";
+
+/// @title OperatorFilterSubription
+/// @notice This contract is ment to register and copy the default subscription of the openSea for the operator filter and our Token contract are supposed to subscribe to This contract on openSea operator filter registry
+/// @custom:experimental This is an experimental contract. There could be future changes according to the change in the requirements
+contract OperatorFilterSubscription is Ownable {
+    address public constant DEFAULT_SUBSCRIPTION = address(0x3cc6CddA760b79bAfa08dF41ECFA224f810dCeB6);
+
+    IOperatorFilterRegistry public constant operatorFilterRegistry =
+        IOperatorFilterRegistry(0x000000000000AAeB6D7670E522A718067333cd4E);
+
+    constructor() Ownable() {
+        // Subscribe and copy the entries of the Default subscription list of open sea.
+        if (address(operatorFilterRegistry).code.length > 0) {
```

Comment:
use openzeppelin isContract
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Address.sol#L40

Reply:
This contract is from the Opensea's npm package, with minimum changes. Should we add that contract to imports ?